tutorials/application.getCurrent.md

```diff
@@ -0,0 +1,17 @@
+Returns an Application object that represents the current application
+
+# Example
+```js
+async function isCurrentAppRunning () {
+    const app = await fin.Application.getCurrent();
+    app.whatever();
```

Comment:
why need whatever() ? Allow users to call any other functions?

Reply:
That's my bad, will, fix it.